packages/webawesome/docs/docs/components/textarea.md

```diff
@@ -6,7 +6,7 @@ category: Form Controls
 ---
 
 ```html {.example}
-<wa-textarea label="Type something', will ya"></wa-textarea>
+<wa-textarea label="Type something, will ya"></wa-textarea>
```

Comment:
This came from 

<img width="768" height="682" alt="image" src="https://github.com/user-attachments/assets/8102f1f6-c17a-4cfb-bddf-935cb88263a5" />

So updating accordingly

```suggestion
<wa-textarea label="Type somethin', will ya"></wa-textarea>
```